phtc/main/tests/test_views.py

```diff
@@ -11,7 +11,7 @@ def setUp(self):
 
     def test_index(self):
         result = self.c.get("/")
-        self.assertEqual(result.status_code, 200)
+        self.assertEqual(result.status_code, 301)
```

Comment:
It looks like 303 and 307 can both be used as temporary redirects: https://en.wikipedia.org/wiki/URL_redirection#HTTP_status_codes_3xx